global/scripts/get_cortex_ROI_in_cifti.m

```diff
@@ -0,0 +1,24 @@
+function [ROI_OUT_CIFTI, NonROI_CIFTI]  = get_cortex_ROI_in_cifti(ROI_label, cifti_parcellation, cifti_space_out)
+    CORTEX_LEFT=cifti_struct_dense_extract_surface_data(cifti_parcellation,'CORTEX_LEFT',1);
+    CORTEX_RIGHT=cifti_struct_dense_extract_surface_data(cifti_parcellation,'CORTEX_RIGHT',1);
+        
+    ROI_CORTEX_LEFT=single(zeros(length(CORTEX_LEFT),1));
+    ROI_CORTEX_RIGHT=single(zeros(length(CORTEX_RIGHT),1));
+    
+    for i=ROI_label
+        ROI_CORTEX_LEFT(CORTEX_LEFT==i)=1;
+        ROI_CORTEX_RIGHT(CORTEX_RIGHT==i)=1;
+    end
+    NonROI_CORTEX_LEFT=(ROI_CORTEX_LEFT-1)*-1;
+    NonROI_CORTEX_RIGHT=(ROI_CORTEX_RIGHT-1)*-1;
+    
+    ROI_OUT_CIFTI=cifti_space_out; % a zero template
```

Comment:
Please actually set the .cdata to an appropriate `zeros(size(..., 1), 1, 'single');` here, rather than trusting that the argument's cdata is both zero and the correct number of maps (1).